run_sim_wecpen.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
i think this script gets really hard to follow because the commands mostly aren't labeled. might be nice to add some check-in #'s to describe what's going on

Reply:
+1

Reply:
Updated.